src/components/About.jsx

```diff
@@ -35,6 +35,48 @@ const About = () => {
         I have created this website to showcase some of my skills, projects and experiences.
         </motion.p>
 
+      <div className="w-full flex row sm:flex-row gap-4 mt-6">
+      <a
+        href="https://github.com/08mfp"
+        target="_blank"
+        rel="noopener noreferrer"
+        className="flex justify-center items-center text-white bg-[#915EFF] px-4 py-2 rounded-full shadow-md hover:bg-purple-700 transition-colors duration-300 w-56 sm:w-auto"
```

Comment:
could later change to use column buttons for website as the 'old website' button goes to 2 lines